tests/cql/CqlIntervalOperatorsTest.xml

```diff
@@ -1016,6 +1016,14 @@
 			<expression>Interval[1, 10] overlaps Interval[4, 10]</expression>
 			<output>true</output>
 		</test>
+		<test name="IntegerIntervalOverlapsTrue2">
+			<expression>Interval[4, 10] overlaps Interval[4, 10]</expression>
+			<output>true</output>
+		</test>
+		<test name="IntegerIntervalOverlapsTrue3">
+			<expression>Interval[10, 15] overlaps Interval[4, 10]</expression>
+			<output>true</output>
+		</test>
```

Comment:
(non blocking thought): would it make sense to add a few tests here (in another PR) for intervals with exclusive boundaries for completeness? it essentially would test that folks are indeed implementing `overlaps` with the proper start/end semantics.

Reply:
Yep, I think that makes sense